docs/config.json

```
@@ -6,7 +6,7 @@
       "nodeIP": "ip-172-31-35-170"
     },
     "access_graph": {
-      "version": "1.28.1"
+      "version": "1.28.0"
```

latest version  is 1.28.1. Fred during our maintenance broke all deployment processes besides teleport.e  
```suggestion
      "version": "1.28.1"
```